TestSwitchApi/TestSwitchApi/Repositories/AdminRepo.cs

```diff
@@ -2,21 +2,36 @@
 using System.Linq;
 using TestSwitchApi.Models.ApiModels;
 using TestSwitchApi.Models.DataModels;
+using TestSwitchApi.Services;
 
 namespace TestSwitchApi.Repositories
 {
     public class AdminRepo : IAdminRepo
     {
         private readonly TestSwitchDbContext _context;
+        private readonly IPasswordService _passwordService = new PasswordService();
 
         public AdminRepo(TestSwitchDbContext context)
         {
             _context = context;
         }
 
+        public void CreateNewAdminUser(string email, string password)
+        {
+            var salt = _passwordService.GenerateSalt();
+            var newAdminUser = new AdminUserDataModel()
+            {
+                Email = email,
+                PasswordSalt = salt,
+                HashedPassword = _passwordService.HashPassword(password, salt),
+            };
+            _context.AdminUsers.Add(newAdminUser);
+            _context.SaveChanges();
+        }
```

Comment:
yeah - much nicer to have it here - would make it easy to add a 'CreateAdmin' endpoint later